packages/one-app-bundler/README.md

```diff
@@ -126,6 +130,19 @@ in this option. Use of this option is at your own risk.
 }
 ```
 
+2. Add `webpackClientConfigPath` and/or `webpackServerConfigPath` to apply different configs to your client and server builds. You don't need to add both of these options. You can pass only one if you wish to customize a single build target.
```

Comment:
we need line breaks here as well

Reply:
done.  it'd be great to have that linted when you commit!